openscap_report/report_generators/html_templates/rule_detail.html

```diff
@@ -136,20 +136,21 @@
             <tr role="row">
                 <th class="pf-m-fit-content" role="rowheader" scope="row"><p class="pf-c-table__text"><b>References:</b></p></th>
                 <td role="cell">
-                    <div class="pf-l-flex pf-m-column"><div class="pf-l-flex__item">
-                    <p class="pf-c-table__text">
-                    <div>
+                    <table>
```

Comment:
Please create a responsive table using CSS classes from Patternfly. Missing CCS classes cause a grey bar. See image.
![image](https://github.com/OpenSCAP/openscap-report/assets/26072444/2c3446d8-bb15-49bd-9ebc-73fcfeb5bc7a)


I think that the rows of the internal table with references should span the full width of the table cell with the OVAL definition. 
![Screenshot from 2023-11-27 19-41-09](https://github.com/OpenSCAP/openscap-report/assets/26072444/921f959c-92de-4bf7-992a-2f67caff47c5)


Example rule: `xccdf_org.ssgproject.content_rule_account_unique_name`

Reply:
Here is Patternfly [documentation](https://v4-archive.patternfly.org/v4/components/table/html). Looks good to me.